batch/test/test_dag.py

```diff
@@ -251,6 +253,12 @@ def test():
         right = batch.create_job('alpine:3.8', command=['echo', 'right'], parent_ids=[head.id])
         tail = batch.create_job('alpine:3.8', command=['echo', 'tail'], parent_ids=[left.id, right.id])
         batch.wait()
+        i = 0
+        while len(output) != 4:
+            j = random.randrange(2 ** i)
+            time.sleep(0.100 * j)
+            if i < 9:
```

Comment:
Would it be better to check if len(output) == 4 within the backoff loop, to break before 9?

Reply:
The loop condition is `len(output) != 4`, so we'll break before 9 if we can.

Reply:
Ah, yep, tired eyes.